packages/checkout-storefront/src/sections/Addresses/utils.ts

```diff
@@ -92,3 +93,41 @@ export const getAddressVlidationRulesVariables = (
         checkRequiredFields: false,
       }
     : {};
+
+export const addressFieldsOrder: AddressField[] = [
+  "firstName",
+  "lastName",
+  "companyName",
+  "phone",
+  "streetAddress1",
+  "streetAddress2",
+  "city",
+  "postalCode",
+  "cityArea",
+  "countryArea",
+];
+
+// api doesn't order the fields but we want to
+export const getOrderedAddressFields = (addressFields: AddressField[] = []): AddressField[] => {
```

Comment:
Threw in something extra for you @mmiszy 😃

Reply:
😆